### PR TITLE
Issue #59: Implement poc ui 

### PR DESCRIPTION
**Pull request by Direside**
Created a web-ui for creating projects.

### CODE REVIEW

File: internal/ui/src/App.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #09d3ac;
+}
```
Comment (zthomas) [inline internal/ui/src/App.css]: Doesn't look like this file is still being used.

File: internal/ui/src/index.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
```
Comment (zthomas) [inline internal/ui/src/index.css]: Is this being used?

File: internal/ui/src/App.css (hunk 1 above)
Comment (Direside) [reply internal/ui/src/App.css]: Just the text-align, moved to index.css

File: internal/ui/src/index.css (hunk 2 above)
Comment (Direside) [reply internal/ui/src/index.css]: Yes, it sets the font for the material ui components.

**Pull request merged**